### PR TITLE
Added git push actions

### PR DESCRIPTION
**Pull request by sanjayradadiya**
## 🧰 Issue

issue #692 

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

File: .github/workflows/serge.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# This workflow will install Deno and run tests across stable and nightly builds on Windows, Ubuntu and macOS.
```
Comment (IanMayo) [inline .github/workflows/serge.yml]: we don't need this comment.

File: .github/workflows/serge.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  push:
+    branches:
+      - master
+      - develop
+
+jobs:
+  test:
+  
+    name: Running unit test cases
+    runs-on: macOS-latest
```
Comment (IanMayo) [inline .github/workflows/serge.yml]: why Macos? I presume GitHub as more  linux machines available.

File: .github/workflows/serge.yml (hunk 1 above)
Comment (sanjayradadiya) [reply .github/workflows/serge.yml]: okay

File: .github/workflows/serge.yml (hunk 2 above)
Comment (sanjayradadiya) [reply .github/workflows/serge.yml]: Correct

**Review by IanMayo — APPROVED** [review]

**Pull request merged**